Documentation/kubernetes-on-aws-render.md

```diff
@@ -0,0 +1,255 @@
+# Configure your Kubernetes cluster on AWS
+
+This is the [second step of running Kubernetes on AWS][aws-step-1]. Before we launch our cluster, let's define a few parameters that the cluster requires.
```

Comment:
Link should enclose only "running Kubernetes on AWS".

It is confusing for 'second step' to be a link to step 1.